workitem/workitem_repository.go

```diff
@@ -512,6 +513,11 @@ func (r *GormWorkItemRepository) Save(ctx context.Context, spaceID uuid.UUID, up
 		}
 		fieldValue := updatedWorkItem.Fields[fieldName]
 		var err error
+		var empty []interface{}
+		if (fieldName == SystemAssignees || fieldName == SystemLabels) && reflect.DeepEqual(fieldValue, empty) {
```

Comment:
do you have to use `reflect.DeepEqual()` ? can't you just cast to `[]interface{}` and get the `len` ? Also, comparing with `nil` would be relevant, IMO.

Reply:
@xcoulon yes, something [like this](https://play.golang.org/p/7n_nIzTBzJ):

```go
a := []interface{}{"alice", "bob"}
	var c interface{} = a
	arr, ok := c.([]interface{})
	fmt.Println(arr, ok)
```

Reply:
Fixed. 1d9f95f53e681addec1d123ed327b585e55f5108

Reply:
ok